### ISSUE TITLE
Fixed downtime that starts in the past never sends DOWNTIMESTART notification

### ISSUE THREAD

**Issue by RobinGeuze** [Icinga/icinga2#7896]
## Describe the bug
If you create a downtime on a service (or host) that starts in the past even slightly it never send the DOWNTIMESTART notification to any configured notification scripts. This is problematic since if you do not change the start time a downtime created via the webinterface will almost always start in the past (since it uses page load as the start time).

## To Reproduce

1. Setup a notification script with at least DOWNTIMESTART enabled
2. Create a downtime which starts slightly into the future (5 seconds)
3. See a notification happen
4. Create a new downtime which start slightly into the past
5. See no notification happen

## Expected behavior

I expect the DOWNTIMESTART to always get triggered.

## Your Environment

* Version used (`icinga2 --version`): r2.11.2-1
* Operating System and version: Debian Stretch
* Enabled features (`icinga2 feature list`):
```
Disabled features: compatlog debuglog elasticsearch gelf graphite opentsdb perfdata statusdata syslog
Enabled features: api checker command ido-mysql influxdb livestatus mainlog notification
```
* Icinga Web 2 version and modules (System - About): 2.7.3

**Comment by Al2Klimov** [comment]
@nilmerg Is this a real-world problem in Icinga Web 2 (any of the monitoring modules)?

**Comment by Al2Klimov** [comment]
@N-o-X If we'd delay the start time to now, IMAO we'd still have a race condition. Could we delay it to (let's say) now+5s?

**Comment by N-o-X** [comment]
@Al2Klimov that would be a possible fix, but it could make things confusing, because someone might want to create a downtime that started in the past for documentation purposes.

**Comment by RobinGeuze** [comment]
IMO the cleanest fix would be to fire a DOWNTIMESTART if a new downtime is created with a starttime in the past (and an endtime in the future). I tried figuring out how to do that but I got a bit lost in the code tbh.

**Comment by wp-perc** [comment]
Hi guys, there is any news about this issue?

It is an awkward problem when integrating Icinga2 with other software via notifications: when someone adds a downtime for now right through the web interface, the DOWNTIMESTART notification is not sent, meaning the other software is not able to know if an object is really in downtime until the downtime itself ends. To have a "perfect" sync between icinga2 and the other system, the only solution seems to be disallowing the use of downtimes from icingaweb2, but is a big restriction...

**Comment by julianbrost** [comment]
This issue sound a lot like a duplicate of #5202 which was fixed by #9131. That fix is also included in stable releases starting from 2.12.7 and 2.13.3. So please try if this is still an issue in a up-to-date version, but I presume is not (but please let us know if it still is).

**Comment by Al2Klimov** [comment]
Duplicate of #5202
